en/setup/dev_env_mac.md

```diff
@@ -3,7 +3,7 @@
 MacOS is a supported development platform for PX4. The following instructions set up an environment for building:
 * NuttX-based hardware (Pixhawk, etc.)
 * jMAVSim Smulation
-* Gazebo 8 Simulation
+* Gazebo Simulation
```

Comment:
It may be important to know the installed Gazebo version, at least somewhere on this page

Reply:
I think it is not in our control really, it's whatever ships by OSRF. For older OSs it might be 7 and for newer ones 9 or 10, so 8 is as wrong as it gets :smile:.

Reply:
Ah, so this isn't a PX4 "choice". OK then.